corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_variables.scss

```diff
@@ -10,6 +10,8 @@ $container-max-widths: (
   xxl: 1160px
 );
 
+// Spacing override
+$enable-negative-margins: true;
```

Comment:
I don't understand what impact will it have 😅

Reply:
Check [this](https://getbootstrap.com/docs/5.0/utilities/spacing/#negative-margin) out: B5's negative margin utility classes are disabled by default, though I don't know why.

So this was needed to use the `mt-n3` added in [this commit](https://github.com/dimagi/commcare-hq/commit/2760a7a0a52398b41a61ea32de9cae9fb6c94f54#diff-6dca3dbb115515abc14b50769fcf871ebe30b05e2f64645034a385a6552eb6ceL41).

Reply:
That makes sense. Thanks for explaining.